lib/wiregasm/lib.cpp

```diff
@@ -4,6 +4,10 @@
 static guint32 cum_bytes;
 static frame_data ref_frame;
 
+// Array of array of strings
+vector<vector<string>> followArray;
```

Comment:
We should be able to get rid of this global var by using the `user_data` pointer that we have access to in `wg_session_follower_visit_cb`.

It is currently used for passing on the `packet_info` pointer. We could just use a struct that holds both of them and pass it on.

Reply:
> We should be able to get rid of this global var by using the `user_data` pointer that we have access to in `wg_session_follower_visit_cb`.
> 
> It is currently used for passing on the `packet_info` pointer. We could just use a struct that holds both of them and pass it on.

It makes sense. The code is updated this way https://github.com/good-tools/wiregasm/pull/3/files#diff-ed806ab4579970b063bc4f6dea27951a12b9c7659683604a45129c387c42a698R617-R624
It's my first experience with cpp, so I'm not sure whether I missed something.

Reply:
Looks good!